code/modules/events/ion_storm.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/ion_storm
 	name = "Ion Storm"
 	typepath = /datum/round_event/ion_storm
-	weight = 15
+	weight = 30 // SKYRAT EDIT - ORIGINAL 15
```

Comment:
This is too high, just leave it as TG default imo

Reply:
:(

Okay fine